iOS Example Frame/iOS Example Frame/Factory/Factory+UITest.swift

```diff
@@ -11,7 +11,13 @@ import Frames
 import Checkout
 
 #if UITEST
-extension Factory {
+enum UITestFactory {
+    // swiftlint:disable:next force_unwrapping
+    static let successURL = URL(string: "https://httpstat.us/200")!
+    // swiftlint:disable:next force_unwrapping
+    static let failureURL = URL(string: "https://httpstat.us/403")!
+    static let apiKey = "pk_test_6e40a700-d563-43cd-89d0-f9bb17d35e73"
+    static let environment: Frames.Environment = .sandbox
```

Comment:
What was gained by this besides adding UITest named objects inside the application source code?